basic/list2.py

```diff
@@ -13,17 +13,28 @@
 # so [1, 2, 2, 3] returns [1, 2, 3]. You may create a new list or
 # modify the passed in list.
 def remove_adjacent(nums):
-    # +++your code here+++
-    return
+    for num in nums:
+        if nums.count(num) > 1:
+            nums.remove(num);
+    return nums;
 
 
 # E. Given two lists sorted in increasing order, create and return a merged
 # list of all the elements in sorted order. You may modify the passed in lists.
 # Ideally, the solution should work in "linear" time, making a single
 # pass of both lists.
 def linear_merge(list1, list2):
-    # +++your code here+++
-    return
+    lista = list()
+    while list1 and list2:
+        if list1[0] < list2[0]:
+            lista.append(list1.pop(0))
+        else:
+            lista.append(list2.pop(0))
+    if list1:
+        lista+=list1
+    if list2:
+        lista+=list2
+    return lista
```

Comment:
szep megoldas, de a pop(0) miatt nem linearis https://stackoverflow.com/questions/195625/what-is-the-time-complexity-of-popping-elements-from-list-in-python